apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -29,10 +37,22 @@ interface IProviderForm {
   identifier: string;
 }
 
+const Info = ({ provider, environments }) => (
```

Comment:
Maybe this should be in its own file even if it is only used here and it feels like this is the only place it will be used it?

Reply:
There is another PR where I think @LetItRock implements something similar. 
https://github.com/novuhq/novu/blob/e97f83e0482882d1b218c9d6cce71e66ff5571b1/apps/web/src/pages/integrations/components/UpdateIntegrationSidebarHeader.tsx#L41

https://github.com/novuhq/novu/pull/3740

So I agree with you and I push for it to be moved it to its own component so it can be reused for you both. 👏🏻